exercises/shared/.docs/representations.md

```diff
@@ -0,0 +1,191 @@
+## JavaScript representations
+
+The JavaScript representer works with the `@typescript-eslint/parser`, a packager that generates an [ESTree](https://github.com/estree/estree).
+
+This abstract syntax tree output is then traversed and the following normalisation steps are performed:
+
+- All comments are ignored / removed
+- All whitespace is ignored
+- All _identifiers_ are renamed, using a "key lookup"
+
+## BEFORE YOU SUBMIT
+
+Before you submit your representation feedback, please check the following things:
+
+1. You don't duplicate analyzer feedback
+2. You check the "examples" tab in the submit dialog and see if the feedback makes sense for _all_ tabs.
+3. You check that you have not referred to whitespace or comments
+4. You check that you don't refer to function names, or variable names as they appear in the solution, but rather use the mapping provided (or leave names out).
```

Comment:
I thought the the mapping is not provided because the UI only shows a solution and never the representation.

Reply:
Correct. But it's supposed to be provided. It's in one of the todos